manifest.json

```diff
@@ -2,16 +2,16 @@
     "spark_version": "2.4.0",
     "default_spark_dist": {
         "hadoop_version": "2.9",
-        "uri": "https://downloads.mesosphere.com/spark/assets/2.8.0/spark-2.4.0-bin-hadoop2.9.2.tgz"
+        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.4.0-bin-2.9.2-openjdk8-SNAPSHOT-20190530.tgz"
```

Comment:
These URIs should not include `SNAPSHOT`

Reply:
Anton told to me add the SNAPSHOT postfix. We remove it eventually once the packages will be moved to production bucket. But Let me know if there is a reason get rid SNAPSHOT now.

Reply:
Yup, sorry about that. Feel free to disregard!

Reply:
no problem :-)